src/search/components/query-editor-modal.html

```diff
@@ -1,11 +1,17 @@
 <div class="modal-header">
-  <h2 class="modal-title">Geavanceerde zoekopdracht bouwer</h2>
+  <h2 class="modal-title" translate-once="search.advancedQueryBuilder.title">Geavanceerde zoekopdracht bouwer</h2>
```

Comment:
The text between the tags can be removed I guess? 
```suggestion
  <h2 class="modal-title" translate-once="search.advancedQueryBuilder.title"></h2>
```